contents/blog/redesigned-nav-menu.mdx

```diff
@@ -0,0 +1,159 @@
+
+- `user:` like `user:marius`
+- `type:` like `type:insight`
+- `name:` like `name:marketing`
+
+It even supports negative keywords so you can exclude items that contain a string, like <code class="text-inherit p-1 rounded bg-accent dark:bg-accent-dark border border-light dark:border-dark !text-[13px] whitespace-nowrap">-this</code>. You can also multi-select items for bulk actions.
+
+<ProductScreenshot
+    imageLight={Filtering}
+    caption="Searching and filtering"
+    padding={false}
+    classes="rounded"
+    alt="Search and filter"
+/>
+
+### Accessibility and keyboard navigation
+
+We're also working toward better accessibility and keyboard navigation across the app. You'll now find it much easier to navigate the menus with tabs, arrows, and enter keys.
+
+<ProductScreenshot
+    imageLight={Accessibility}
+    caption="Accessibility and keyboard navigation"
+    padding={false}
+    classes="rounded"
+    alt="Accessibility and keyboard navigation"
+/>
+
+### Expanded view
+
+Since files are now in a sidebar column, some metadata is hidden by default (like who created an item and when it was created). So we added an expanded view that overlays the sidebar on the page to see more details.
+
+<ProductScreenshot
+    imageLight={ExpandedLight}
+    imageDark={ExpandedDark}
+    caption="Expanded view"
+    padding={false}
+    classes="rounded"
+    alt="Expanded view"
+/>
+
+## One more thing... shortcuts!
+
+Until now, the navigation menu has been static. If you only used a handful of PostHog product, too bad! You still had to look at the entire list of products.
+
+We're also rolling out customizable shortcuts which will let you pin just about anything to the main navigation sidebar. This includes products, specific product features, folders, or individual items like insights, dashboards, or experiments. It's a great way to keep tabs on what's important to you right now.
+
+Below you'll see the products menu replaced with shortcuts.
+
+<ProductVideo
+    videoLight= "https://res.cloudinary.com/dmukukwp6/video/upload/Shortcuts_acfb3a0ae8.mp4" 
+    alt="Organizing content in folders" 
+    classes="rounded shadow-xl"
+    autoPlay={false}
+/>
+
+We'll soon replace the products menu with shortcuts, but you can try it now by clicking the gear icon and selecting _Shortcuts_ in the modal.
+
+### Products menu
+
+The full list of products are now available in the new searchable _Products_ menu, always available directly from the main navigation.
+
+<ProductScreenshot
+    imageLight={ProductsLight}
+    imageDark={ProductsDark}
+    caption="Folders let you organize your content in PostHog"
+    padding={false}
+    classes="rounded"
+    alt="Folders"
+/>
+
+We're currently rolling out this new navigation design and, very soon, will be defaulting to showing shortucts instead of the ever-growing list of products. Until then, we hope you enjoy the newly polished sidebar and find it easier to navigate PostHog.
```

Comment:
```suggestion
We're currently rolling out this new navigation design and, very soon, will be defaulting to showing shortcuts instead of the ever-growing list of products. Until then, we hope you enjoy the newly polished sidebar and find it easier to navigate PostHog.
```